monitor.js

```diff
@@ -0,0 +1,117 @@
+const fetch = require('node-fetch')
+const config = require('./monitor.json');
+var env=null;
```

Comment:
Use let instead of var
Additionally, I believe these variables should be declared inside the functions and passed through them